server/src/utils/getConfig.test.ts

```diff
@@ -288,4 +291,88 @@ describe('config', () => {
     expect(getDefaultConfigSpy).toHaveBeenCalled();
     expect(config.postCall).not.toBeDefined();
   });
+
+  test('getSeverConfig returns undefined cosmosDb without any cosmosDb', () => {
+    const config = getConfig.getServerConfig();
+
+    expect(config.cosmosDb).toBeUndefined();
+  });
+
+  test('getServerConfig returns cosmosDb with connection string with database name from environment variable', () => {
+    process.env.VV_COSMOS_DB_CONNECTION_STRING = 'testingConnectionString';
+    process.env.VV_COSMOS_DB_NAME = 'testingDBName';
+
+    const config = getConfig.getServerConfig();
+
+    expect(config.cosmosDb).not.toBeUndefined();
```

Comment:
`.not.toBeUndefined()` I think we can change this to `.toBeDefined()` ?

Reply:
I think either way is fine to me. I will update it to `toBeDefined()`.